### ISSUE TITLE
`logits_real = logits_fake.reshape(B, -1)`: is this on purpose or just a bug?

### ISSUE THREAD

**Issue by AIWanderer-X** [TencentARC/Open-MAGVIT2#20]
https://github.com/TencentARC/Open-MAGVIT2/blob/6bfab153fb2fe612edbf8afa9b3a607d8337c194/taming/modules/losses/vqperceptual.py#L58C5-L58C16

**Comment by RobertLuo1** [comment]
Hi, Thanks for your pointing out. It is indeed a bug that may be caused when I merge the code. However, we have not utilized that loss when training the discriminator. It won't make any impact on the Tokenizer training.
